.npmignore

```diff
@@ -1,3 +1,37 @@
-/example
-/src
-circle.yml
+# Tests
+e2e
+example
+
+# sources are inlined
+src
+
+# Developement scripts
+scripts
+
+# Coverage directory used by tools like istanbul
+coverage
+
+# nyc test coverage
+.nyc_output
```

Comment:
Do we have nyc test coverage?

Reply:
Oh this we don’t need, copy paste mistake 😅

Reply:
done